### PR TITLE
Show reviews

### PR DESCRIPTION
**Pull request by Egoes98**
The methods and GUI for showing the reviews. When a guest searches for a hotel and enters the view where rooms are shown there appears a button that show a GUI dispalying the Reviews teh hotel has and also the average score.

### CODE REVIEW

File: src/test/java/es/deusto/spq/server/data/dto/AssemblerTest.java [hunk 1]
```diff
@@ -226,12 +226,10 @@ public void assembleReview_Review_Assemble() {
                 "testreview",
                 "opinion here",
                 7,
-                timestamp,
-                assembler.assembleHotel(hotel),
-                assembler.assembleUser(guest)
+                timestamp
         );
 
-        Assert.assertEquals(reviewDTO, assembler.assembleReview(review));
```
Comment (iapellaniz) [inline src/test/java/es/deusto/spq/server/data/dto/AssemblerTest.java]: Why this change?

File: src/main/java/es/deusto/spq/client/gui/views/reviews/ShowHotelReviewView.java [hunk 2]
```diff
@@ -0,0 +1,181 @@
...
+		lblTittle.setBounds(191, 11, 85, 25);
+		frame.getContentPane().add(lblTittle);
+		
+		//Displays the Average Score
+		averageScore = 0;
+		lblAverageScore = new JLabel(getViewManager().getClient().getLocaleManager().getMessage("showReviews.averageScore")+" "+averageScore, JLabel.TRAILING);
+		lblAverageScore.setBounds(191, 47, 85, 25);
+		frame.getContentPane().add(lblAverageScore);
+		
+		frame.setVisible(true);
```
Comment (iapellaniz) [inline src/main/java/es/deusto/spq/client/gui/views/reviews/ShowHotelReviewView.java:140]: Missing `addDisposeEventHandler` method call at the end of initialization.

**Review by iapellaniz — CHANGES_REQUESTED** [review]

File: src/test/java/es/deusto/spq/server/data/dto/AssemblerTest.java (hunk 1 above)
Comment (Egoes98) [reply src/test/java/es/deusto/spq/server/data/dto/AssemblerTest.java]: yeah sorry i will put it back as it was, it is becaus ei changed the dto and at first i thought i should change that also but then i realized that wasnt the case.

**Review by iker-barriocanal — DISMISSED** [review]
Would be useful to have an example review to see it.

**Comment by Egoes98** [comment]
I have added a file with the SQL needed to create the example data. You have to create the DB and then execute this SQL statements.

**Comment by iapellaniz** [comment]
Tests are failing due to performance issues. Maybe we should increase the limit to 300ms

**Comment by Egoes98** [comment]
> Tests are failing due to performance issues. Maybe we should increase the limit to 300ms

Could be a good idea, i will change it.

**Review by iapellaniz — APPROVED** [review]
